### ISSUE TITLE
Figure out sizes and content wanted in each

### ISSUE THREAD

**Issue by bfred-it** [bfred-it/antivaccinebodycount#1]
I made a page with the counters from your site, so you won't have to update them manually. It's visible here: http://bfred-it.github.io/antivaccinebodycount/examples.html

On your computer (no touch) you can play with each box by dragging each bottom-right corner and see how they look at different sizes.

Do you know who was using them and the size they'd like?

Also, what content/text do you want to show in them, if any?

**Comment by derekcbart** [comment]
Hello.

I really like the layouts. That's very cool that you can adjust the sizes. 

The issues that have happened in the past had to do with automatically updating of the numbers. Everything is basically built in text in iWeb. iWeb has been discontinued and I am planning to convert the website to EverWeb, but I do not know when that conversion will be finished.

In the past I would update the website and sometimes the widget would update and sometimes it would not. I don't know anything about how widgets are created, but I do not have the ability to edit the HTML of the page directly, so that limited the ability to create widgets.

Thanks.

-Derek

**Comment by bfred-it** [comment]
Those numbers will be automatically updated instantly and as the are pulled straight from your site, always. The only issue is that they highly depend on your homepage: if you make substantial changes (like add images above the numbers or change the order of the numbers) they might break. This can be easily fixed by having you update the numbers manually on the widget too. 

How did the previous widget work? Did it take the numbers from your site automatically? 

Anyway, if you're going to move away from iWeb I very much suggest giving [RapidWeaver]( a try. EverWeb seems very amateurish.

**Comment by derekcbart** [comment]
Yes, iWeb and EverWeb are easier. I own DreamWeaver, but the simplicity is what I like because I am not someone who can create websites from scratch. All of my websites are based on templates and iWeb offered a better looking template than anything I could find for DreamWeaver. Of course, that caused limitations in customization. EverWeb is designed to convert iWeb projects, so that is why I am moving towards that. EverWeb will also provide mobile versions, which iWeb cannot.

I have no idea how the previous widget was created nor how it pulled the numbers from the homepage.

Thanks.

-Derek

**Comment by bfred-it** [comment]
RAPIDweaver 😉 
It's a similar software to iWeb. Dreamweaver is something else I don't suggest to anyone 😃
